ReadMe.md

```diff
@@ -136,6 +136,17 @@ For the Asp.NET projects, including the core library, you can run on the latest
 
 For the Maui sample project, you need the latest [_preview_ of Visual Studio](https://visualstudio.microsoft.com/vs/preview/).
 
+If you have not installed node.js on your system, you will need to download and install it in order for the npm scripts to run. Please select
```

Comment:
I made a few grammatical changes, and cleaned up some links to tie them to text. Also, consider using `backtick` to mark off code variables, namespaces, etc. in markdown.

Reply:
good adjustments